getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -84,7 +92,7 @@ Set up a target
 	# Print your currently booted (and known-safe) option
 	cat /proc/cmdline
 	# List the grub entries
-	awk '/^menuentry/ { print i++, '\t', $0 }' /boot/grub/grub.cfg
+	awk '/^menuentry|submenu/ { print i++, '\t', $0 }' /boot/grub/grub.cfg
```

Comment:
there should not be any submenu at this point?

But `sudo` might be needed to actually read the grub.cfg

Reply:
> there should not be any submenu at this point?

Not in theory.

In practice this "magic" command is typically re-used across devices - including devices that still have submenus for whatever reason. And the consequences can be deadly.

> But sudo might be needed to actually read the grub.cfg

In my experience it's not; I use this command all the time.